modules/tasks/tasks/src/main/java/org/motechproject/tasks/service/impl/TaskInitializer.java

```diff
@@ -104,4 +108,9 @@ private Object getDataSourceObject(DataSource dataSource, Map<String, DataProvid
 
         return provider.lookup(dataSource.getType(), dataSource.getName(), lookupFields);
     }
+
+    private Object getPostActionParameterObject() {
```

Comment:
This method would be helpful if we implemented a postActionParameters as task config step. I gave up that idea and removed this method.